eng/common/scripts/Delete-Remote-Branches.ps1

```diff
@@ -0,0 +1,44 @@
+param(
+  $RepoOwner,
+  $RepoName,
+  $BranchPrefix,
+  $WorkingDirectory,
+  $AuthToken
+)
+
+. "${PSScriptRoot}\logging.ps1"
+. "${PSScriptRoot}\Invoke-GitHub-API.ps1" -AuthToken $AuthToken
+
+pushd $WorkingDirectory
+git clone https://github.com/$RepoOwner/$RepoName.git
+pushd $RepoName
+$syncBranches = (git branch --remote).where{ $_ -like "*origin/$BranchPrefix*" }
+
+LogDebug "Operating on Repo [ $RepoName ]"
+foreach ($branch in $syncBranches)
+{
+  try {
+    $branchName = ($branch.Trim()).Replace("origin/","")
+    $head = "${RepoOwner}/${RepoName}:${branchName}"
+    LogDebug "Operating on branch [ $branchName ]"
+    $response = ListPullRequests -RepoOwner $RepoOwner -RepoName $RepoName -head $head
+  }
+  catch
+  {
+    LogError "ListPullRequests failed with exception:`n$_"
```

Comment:
We need to also have a plan for orphaned pull requests where the tools PR has been closed or merged.